docs/toc.yml

```
@@ -312,6 +312,9 @@ items:
       href: community-toolkit/hosting-data-api-builder.md
     - name: EventStore
       href: community-toolkit/hosting-eventstore.md
+  - name: Use Entity Framework Core for database interaction
+    href: database/use-entity-framework-db-contexts.md
```

This should just say entity framework

```suggestion
  - name: Entity Framework
    href: database/use-entity-framework-db-contexts.md
```

I think we could also consider nesting all of the entity framework specific provider docs under here. Treat it like azure.

Thoughts @IEvangelist

---

That might work, but it might also make it harder to find non-EF core content since we can only link to an article once in the TOC. Today, we use that provider type as a parent node, and nested under it the EF and non-EF integrations. That would really change the TOC, but I tend to agree that we should group the EF content together. Let's try it out, we can always change it later.

---

Great.